bin/configurator/static/_includes/ru/configurator/tab/ci/argocd-with-gitlab-ci-cd/simple/docker-runner/linux/buildah/project.md.liquid

```diff
@@ -0,0 +1,83 @@
+## Настройка приложения ArgoCD
+
+1. В целевом кластере примените следующий Application CRD, чтобы развернуть бандл из реестра контейнеров:
+
+```shell
+kubectl create -f - <<EOF
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  annotations:
+    argocd-image-updater.argoproj.io/chart-version: ~ 1.0
+    argocd-image-updater.argoproj.io/pull-secret: pullsecret:myproject-production/myproject-regcred
+  name: myproject
+  namespace: argocd
+  finalizers:
+  - resources-finalizer.argocd.argoproj.io
+spec:
+  destination:
+    namespace: myproject-production
+    server: https://kubernetes.default.svc
+  project: default
+  source:
+    chart: myproject
+    repoURL: registry.mycompany.org/myproject
+    targetRevision: 1.0.0
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
+EOF
+```
+
+Значение `argocd-image-updater.argoproj.io/chart-version="~ 1.0"` означает, что оператор должен автоматически развернуть чарт с последней версией патча в диапазоне `SEMVER` `1.0.*`.
+
+2. Создайте секрет для доступа к реестру контейнеров проекта:
```

Comment:
> секрет

Are we writing in Russian now too?

Reply:
I've no idea :) It looks like we do. Waitin' for @alexey-igrychev on this as well.

Reply:
I suggest using "Secret" as in the guide's articles.